engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -887,4 +891,13 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles): Future[WorkflowDescriptor] = {
```

Comment:
Oh wait, I get it now.

If it's coming from the DB shouldn't we know that the values are actually valid already?